roles/cis/tasks/5/5.1.3.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "5.1.3 Ensure permissions on /etc/cron.hourly are configured (Scored)"
+  file:
+    path: "/etc/cron.hourly"
+    owner: root
+    group: root
+    mode: 0700
```

Comment:
Is this 600 ? The spec says 600, please check

Reply:
Changes done.